rts/Lua/LuaVBOImpl.cpp

```diff
@@ -499,6 +596,16 @@ size_t LuaVBOImpl::Upload(const sol::stack_table& luaTblData, sol::optional<int>
 	return UploadImpl<lua_Number>(dataVec, elemOffset, attribIdx);
 }
 
+
+/***
+ *
+ * @function VBO:Download
+ * @number[opt] attributeIndex
+ * @number[opt] elementOffset
+ * @number[opt] elementCount
+ * @bool[opt] forceGPURead
```

Comment:
Forces downloading the data from GPU buffer as opposed to using shadow RAM buffer.

Reply:
Default `false`?

Reply:
Yes, default is `false`